cla_backend/apps/reports/views.py

```diff
@@ -193,31 +201,11 @@ def download_file(request, file_name="", *args, **kwargs):
     response["Content-Disposition"] = "attachment; filename=%s" % smart_str(file_name)
     response["X-Sendfile"] = smart_str("%s%s" % (settings.TEMP_DIR, file_name))
 
-    try:
-        export_record = Export.objects.get(user_id=request.user.pk, path__endswith=file_name)
-        export_record.delete()
-    except Export.DoesNotExist:
-        raise Http404("Export does not exist")
+    if not file_name.__contains__("scheduled"):
```

Comment:
Shouldn't have to use the double underscores on both sides ("dunder") methods directly.
They're there to define the behaviour of python's keyword syntax.

Is there a specific reason this cannot be `if "scheduled" not in file_name`?

Reply:
the specific reason would be : brain fog.  Quite literally dont know why i went this route.